### PR TITLE
Serviceshub/assessments

### PR DESCRIPTION
**Pull request by pranatiy0701**
## PR Checklist

I've added workbook templates for each type of assessment we have on ServicesHub. I have also added the corresponding gallery for the templates.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
portal view for serviceshub workbook
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: gallery/serviceshub/Assessments.json
Comment (gardnerjr) [inline gallery/serviceshub/Assessments.json:1]: the existance of this file in a new folder "serviceshub" implies that there's a workbook type "serviceshub" but there isn't.  creation of new workbook types requires workbooks team to do work to create those types, is this file expected to be in the "workbooks" folder instead?

**Review by gardnerjr — CHANGES_REQUESTED** [review]

File: gallery/serviceshub/Assessments.json
Comment (pranatiy0701) [reply gallery/serviceshub/Assessments.json]: Worked on and made changes to code - as per our discussion in the email.

**Comment by pranatiy0701** [comment]
> there's also no CODEOWNERS file entries here to mark these new areas as owned by your team.

**Comment by pranatiy0701** [comment]
> there's also no CODEOWNERS file entries here to mark these new areas as owned by your team.

I've added the team's name in ownership.


> > there's also no CODEOWNERS file entries here to mark these new areas as owned by your team.

**Comment by gardnerjr** [comment]
FYI, you don't need screenshots of *code*, as the code is already here in the PR.  we're looking for screenshots that show your templates in the gallery (to prove to yourself and others that they appear where you expect them)

File: gallery/workbook/microsoft.operationalinsights-workspaces.json
Comment (gardnerjr) [inline gallery/workbook/microsoft.operationalinsights-workspaces.json:148]: FYI: you don't need "workbook template' in the name of every item.  that's like naming all your word documents "resume word document.docx"

File: CODEOWNERS
Comment (gardnerjr) [inline CODEOWNERS:160]: do you really want this one?  in this part you're now claiming that you *own* this gallery and can review+approve any changes to this gallery?

**Review by gardnerjr — APPROVED** [review]
i haven't looked at the individual templates as i don't know that i have any workspaces that they'll work on?

all of the templates are marked isPreview so they won't show up by default anyway, so that's fine with me.

but someone from your team should review *every individual template*, that they work well in the happy path *and* in workspaces that do *not* have tables you might need and are following all the best practices around showing useful info/upsell if tables they depend on are missing, etc.

File: CODEOWNERS
Comment (pranatiy0701) [reply CODEOWNERS:160]: I assumed it was part of taking responsibility to the changes I made there. Do you suggest otherwise?

**Comment by pranatiy0701** [comment]
> i haven't looked at the individual templates as i don't know that i have any workspaces that they'll work on?
> 
> all of the templates are marked isPreview so they won't show up by default anyway, so that's fine with me.
> 
> but someone from your team should review _every individual template_, that they work well in the happy path _and_ in workspaces that do _not_ have tables you might need and are following all the best practices around showing useful info/upsell if tables they depend on are missing, etc.

My teammate Rashika has reviewed the changes, and I've tested them individually by copying the template code and pasting it in the Advance tools.

**Review by gardnerjr — APPROVED** [review]
i see this in the templates, where if the table is empty, so nothing is selected, there's a step below that probably should be hidden unless something is selected:

image

this should get fixed in future versions, otherwise users might submit bugs/feedback that something is missing or broken since they might  not understand the message

**Review by eternovsky — APPROVED** [review]
Approving per information provided in the conversation, and with all workbooks being "preview=true".

**Pull request merged**
